Formula/symengine.rb

```diff
@@ -0,0 +1,51 @@
+class Symengine < Formula
+  desc "Fast symbolic manipulation library written in C++"
+  homepage "https://sympy.org"
+  url "https://github.com/symengine/symengine/releases/download/v0.8.1/symengine-0.8.1.tar.gz"
+  sha256 "41eb6ae6901c09e53d7f61f0758f9201e81fc534bfeecd4b2bd4b4e6f6768693"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  depends_on "flint"
+  depends_on "gmp"
+  depends_on "libmpc"
+  depends_on "mpfr"
+
+  def install
+    system "cmake", "-S", ".", "-B", "build",
+                    "-DBUILD_SHARED_LIBS=ON",
+                    "-DWITH_GMP=ON",
+                    "-DWITH_MPFR=ON",
+                    "-DWITH_MPC=ON",
+                    "-DINTEGER_CLASS=flint",
+                    "-DWITH_LLVM=ON",
+                    "-DWITH_COTIRE=OFF",
+                    "-DLLVM_DIR=#{Formula["llvm"].opt_lib}/cmake/llvm",
```

Comment:
This needs to be `llvm@12` if you need LLVM 12. You're also missing a
```ruby
depends_on "llvm@12`
```

Reply:
Sorry, I mean with the fix it builds with LLVM 13, I also added the dependency.